docs/configuration.md

```diff
@@ -57,21 +57,39 @@ Property
-**envoy-control.envoy.snapshot.clusterOutlierDetection.enforcingSuccessRate**               | The % chance that a host will be actually ejected when an outlier status is detected through success rate statistics                                                                      | 100
-**envoy-control.envoy.snapshot.clusterOutlierDetection.interval**                           | The time interval between ejection analysis sweeps                                                                                                                                        | 10s
-**envoy-control.envoy.snapshot.clusterOutlierDetection.maxEjectionPercent**                 | The maximum % of an upstream cluster that can be ejected due to outlier detection                                                                                                         | 10
-**envoy-control.envoy.snapshot.clusterOutlierDetection.successRateMinimumHosts**            | The number of hosts in a cluster that must have enough request volume to detect success rate outliers                                                                                     | 5
-**envoy-control.envoy.snapshot.clusterOutlierDetection.successRateRequestVolume**           | The minimum number of total requests that must be collected in one interval (as defined by the interval duration above) to include this host * in success rate based outlier detection    | 100
-**envoy-control.envoy.snapshot.clusterOutlierDetection.successRateStdevFactor**             | This factor is used to determine the ejection threshold for success rate outlier ejection.                                                                                                | 1900
+**envoy-control.envoy.snapshot.load-balancing.weights.enabled**                             | if set to true, weighted load balancing will be enabled                                                                                                                                   | false
+**envoy-control.envoy.snapshot.load-balancing.canary.enabled**                              | if set to true, routing to canary instances based on *canary header* will be enabled (corresponding Envoy static config is required, see [docs](../features/load_balancing))              | false
+**envoy-control.envoy.snapshot.load-balancing.canary.metadata-key**                         | metadata that will be set for canary EDS endpoints - key (must match Envoy static `header_to_metadata` filter config, see [docs](../features/load_balancing))                             | canary
+**envoy-control.envoy.snapshot.load-balancing.canary.header-value**                         | only when *canary header* is set to this value request will be routed to canary instances (*canary header* name is set in Envoy static config, see [docs](../features/load_balancing))    | 1
+
+## Routing
+Property                                                                                    | Description
```

Comment:
You can change format to same os for other properties: additional column `Default value`

Reply:
fixed